interfaces/Libraries.ts

```diff
@@ -116,6 +116,11 @@ from pyannote.core import Segment
 excerpt = Segment(start=2.0, end=5.0)
 model.crop("file.wav", excerpt)`;
 
+const tensorflowtts = (model: ModelData) =>
+`from tensorflow_tts.inference import TFAutoModel
+model = TFAutoModel.from_pretrained("${model.modelId}")
```

Comment:
Should it also showcase the `AutoProcessor` loading?

Reply:
The `AutoProcessor` only makes sense in the context of models that convert text to mel spectogram. In the context of models that convert mel to wav, there are no `AutoProcessor`s.  On the other hand, text to mel will be the most common use case. 

What do you think of this?
* We use tags `text-to-mel` and `mel-to-speech`. Based on this, we change the code snippet (just as espnet).
* This will probably be useful once we implement the inference API since the code will also differ between these two.